### PR TITLE
Add a test case to find the maximum no of queries 

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Fixes: [1889](https://github.com/ballerina-platform/ballerina-standard-library/issues/1889)

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/tests/batch-execute-query-test.bal [hunk 1]
```diff
@@ -96,6 +100,21 @@ function batchInsertIntoDataTableFailure3() {
...
+function batchInsertIntoDataTable4() returns error? {
+    string path = check file:getAbsolutePath("tests/resources/sample/parameterized_query.bal");
+    Process process = check exec("bal", {}, (), "run", path);
+    int waitForExit = check process.waitForExit();
+    int exitCode = check process.exitCode();
+    io:ReadableByteChannel readableResult = process.stderr();
+    io:ReadableCharacterChannel sc = new (readableResult, "UTF-8");
+    string outText = check sc.read(100000);
+    string[] texts = regex:split(outText, "\n");
+    test:assertTrue(strings:includes(texts[3], "method is too large"));
```
Comment (niveathika) [inline ballerina/tests/batch-execute-query-test.bal:115]: Is this a negative test?

Comment (kalaiyarasiganeshalingam) [reply ballerina/tests/batch-execute-query-test.bal:115]: Yes, this is a negative test. I have added a `negative` suffix in c0db6a8.

Comment (niveathika) [reply ballerina/tests/batch-execute-query-test.bal:123]: Do we need a negative test here? Out object is this error should not be thrown right?

File: ballerina/tests/batch-execute-query-test.bal [hunk 2]
```diff
@@ -96,6 +99,30 @@ function batchInsertIntoDataTableFailure3() {
     test:assertTrue(result is ApplicationError);
 }
 
+@test:Config {
+    groups: ["batch-execute"]
+}
+function batchInsertNegative() returns error? {
+    string path = "../ballerina/tests/resources/sample/parameterized_query.bal";
+    Process process = check exec("ballerina", {}, (), "version");
```
Comment (niveathika) [inline ballerina/tests/batch-execute-query-test.bal:107]: Ballerina is not installed in the machine. You need to call the ballerina.bat script from the target folder.

Comment (kalaiyarasiganeshalingam) [reply ballerina/tests/batch-execute-query-test.bal]: Removed this in 2406fa6

File: ballerina/tests/batch-execute-query-test.bal (hunk 1 above)
Comment (kalaiyarasiganeshalingam) [reply ballerina/tests/batch-execute-query-test.bal]: Changed to positive test in 2406fa6

**Review by daneshk — APPROVED** [review]
